packages/projects-docs/pages/learn/repositories/overview.mdx

```diff
@@ -27,3 +27,16 @@ When you press `Create Branch`, we automatically create a new branch and set up
 <Video src="../../repo-branching.mp4" />
 
 If you are looking at someone’s PR and want to quickly test a suggestion, simply click `Create Branch`, write the code and share a link to your changes. If you’re happy with the changes, merge that into the existing PR. All of this in under a minute.
+
+## Understanding repository imports
+
+CodeSandbox handles imported repositories on a per-workspace basis. As a result, any given repository may exist in different states if it has been imported to multiple CodeSandbox workspaces. Additionally, the public version of a repository will always have its own, independent state.
```

Comment:
Instead of leading with the idea that repos can exist in different states, why don't we just explain how it works? That way, users don't have to ask what the implications are or how to manage multiple states
```suggestion
CodeSandbox handles imported repositories on a per-workspace basis. This means that the changes made to a repository in one workspace will only be visible to users within that workspace until those changes are synced with GitHub. 
```